nyc.config.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   "check-coverage": true,
   all: true,
   include: "**/src/**",
+  exclude: "**/src/extension.ts",
```

Comment:
I think this exclusion does not belong in the **root** config
rather in the specific sub-package config as not all sub-packages have an extension.ts

Reply:
Done